### ISSUE TITLE
No SIFT scores for reference alleles

### ISSUE THREAD

**Issue by Melanie-Wilkinson** [pauline-ng/SIFT4G_Annotator#11]
Why does SIFT ignore the reference allele in its output? All my dbSNP are novel and therefore there are no "ref" alleles that have SIFT scores i.e only the "alt" alleles based on the reference genome have SIFT scores. This occurs when I run a single or multiple individuals through the SIFT4G Annotator. 

I tried to overcome this by using the following but still no ref alleles are output:
#trim alternate alleles not seen in the subset and exclude sites without a called genotype
bcftools view --trim-alt-alleles --exclude-uncalled

It looks like SIFT is just matching the "Ref" and "Alt" columns in the vcf file and ignoring the genotypes.

**Comment by pauline-ng** [comment]
That's right, it ignores genotypes. To get the SIFT prediction for a ref allele, put the ref allele in the alt column.

**Comment by Melanie-Wilkinson** [comment]
Thanks, Pauline. I think SIFT isn't recognising any of my monomorphic sites because instead of the ref allele in my ALT column, my vcf files have '.'
